lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
Lionel,

considering the set of emails following the one I am responding to, I think
I
will stay with my original plan and make dde for Sgui work correctly on my
local copy.

I am naive on git so I won't attempt to upload until later.
Please send me the line I need to download a clean copy of whatever branch
you want me to start with.  Once it works locally, then I will ask for the
next set of instructions to upload and merge it.

Rich

On Sun, Apr 3, 2016 at 11:59 PM, Lionel Henry <notifications@github.com>
wrote:

> In lisp/ess-custom.el
> <https://github.com/emacs-ess/ESS/pull/309#discussion_r58326608>:
>
> > @@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
> >
> >  ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
> >
> > -(defcustom inferior-ess-ddeclient nil
> > +(defcustom inferior-ess-ddeclient "Initial"
>
> Thanks Richard. Could you please make those changes in the developer2
> branch rather than on master, so that we don't have to do the work twice.
>
> Is ess-rgui.el still relevant at all for using ESS on Windows? If it's
> possible to interact with the R process the normal way, I think we should
> drop support for RGUI interaction, especially considering the limitations
> you just exposed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/emacs-ess/ESS/pull/309/files/603f35f77e2039f5fea8c52a643347b70f77c4be#r58326608>
>